core/iwasm/common/arch/invokeNative_riscv.S

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (C) 2019 Intel Corporation.  All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ */
+
+#if __riscv_xlen == 32
+#define RV_OP_LOADREG  lw
+#define RV_OP_STOREREG sw
+#define RV_REG_SIZE    4
+#define RV_REG_SHIFT   2
+#else
+#define RV_OP_LOADREG  ld
+#define RV_OP_STOREREG sd
+#define RV_REG_SIZE    8
+#define RV_REG_SHIFT   3
+#endif
+
+#ifdef __riscv_float_abi_single
```

Comment:
Does riscv spec define that these macros must be defined by compiler by default if float abi is enabled? If yes, we can use them, if not, seems that we should use BUILD_TARGET_RISCV32/64_XXX instead.

Reply:
It works on GCC, but I'm not sure how it works on other platform  (clang?)

Reply:
I checked the [riscv-c-api](https://github.com/riscv/riscv-c-api-doc/blob/master/riscv-c-api.md) document, seems that c compiler should define these macros accordingly. So could you add a comment/note in the beginning of the file to claim that all the following macros refer to that document?